.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  # Enable version updates for nuget
+  — package-ecosystem: “nuget”
+  # Look for NuGet dependency info from the `root` directory
+  directory: “/”
+  # Check the nuget registry for updates every day (weekdays) schedule:
+  interval: “daily"
```

Comment:
Keep the interval as weekly. I noticed that one of our projects (Osmo-Notify) has a weekly check.

Reply:
Changed this to weekly